CoreScriptsRoot/CoreScripts/Topbar.lua

```diff
@@ -952,9 +973,13 @@ local function CreateLeaderstatsMenuItem()
 		PlayerlistModule.TopbarEnabledChanged(enabled and not InputService.VREnabled) --We don't show the playerlist at all in VR
 	end)
 
+	local Utility = require(GuiRoot:WaitForChild("Modules"):WaitForChild("Settings"):WaitForChild("Utility"))
```

Comment:
Should we just require this module at the top of the script since it is required at multiple places throughout the script.